mobile-app/app/screens/AppNavigator/screens/Loans/hooks/CollateralizationRatioV2.ts

```diff
@@ -0,0 +1,209 @@
+import { ThemedProps } from "@components/themed";
```

Comment:
There isn't any modification from `CollateralizationRatio.ts`, but 2 added funcs. Let's not create a v2 to reduce refactoring on Vault Detail screen